app/assets/stylesheets/modules/_header-tweaks.scss

```diff
@@ -0,0 +1,39 @@
+// We have both a title and a search box in our header. This is unusual. To make
+// this work, we need to slightly tweak the header's CSS from our app.
+.header-title {
+  float: left;
+  color: $white;
+  @include bold-24;
+  
+  // When the title appears on its own line (the search box is hidden and only a
+  // search toggle is visible) it needs some padding
+  @media screen and (max-width: 379px) {
+    margin: 5px 15px 0;
+  }
+
+  // When it appears next to the crown, align the heading with the baseline of
+  // the GOV.UK logo...
+  // ...at 16px
+  @media screen and (min-width: 380px) {
+    margin-top: 14px;
```

Comment:
My only suggestion would be to assign it to a variable with an comment like name. An extreme example would be `$difference-in-height-between-title-and-the-logo = 14px;` or something, and remove the comment. That _might_ be better.. but only just if so. I'd probably leave it.